qrenderdoc/Windows/Dialogs/CaptureDialog.cpp

```diff
@@ -397,6 +426,74 @@ void CaptureDialog::vulkanLayerWarn_mouseClick()
   }
 }
 
+void CaptureDialog::androidLayerWarn_mouseClick()
+{
+  QString exe = ui->exePath->text();
+
+  QString caption = tr("Missing RenderDoc layer");
+
+  QString msg =
+      tr("In order to debug on Android, the RenderDoc layer must be present in the "
+         "installed application.<br><br>"
+         "To fix this, you should repackage the APK following guidelines on the "
+         "<a href='http://github.com/baldurk/renderdoc/wiki/Android-Support'>"
+         "RenderDoc Wiki</a><br><br>"
+         "If you are only targeting Vulkan, RenderDoc can try to add the layer for you, "
+         "which requires pulling the APK, patching it, and reinstalling with a debug key. "
+         "This works for many debuggable applications, but not all, especially those that "
+         "check their integrity before launching.<br><br>"
+         "Your system will need Android Studio with several subcomponents installed for "
```

Comment:
No, it isn't required, I'll update the text.  It is just the easiest way to get all the dependencies.

Although the IDE does create the debug keystore I'm checking for in default location.  If it isn't there, we can certainly create it using keytool.  I might do that as a follow up commit instead of adding it to this one.